database/seeds/SourceServiceTableSeeder.php

```diff
@@ -192,13 +192,58 @@ private function addCoreOptions()
             'script_entry' => 'bash',
             'script_container' => 'ubuntu:16.04',
         ]);
+
+        $script = <<<'EOF'
+#!/bin/bash
+# CSGO Installation Script
+#
+# Server Files: /mnt/server
+apt -y update
+apt -y --no-install-recommends install curl lib32gcc1 ca-certificates
+
+cd /tmp
+curl -sSL -o steamcmd.tar.gz http://media.steampowered.com/installer/steamcmd_linux.tar.gz
+
+mkdir -p /mnt/server/steamcmd
+tar -xzvf steamcmd.tar.gz -C /mnt/server/steamcmd
+cd /mnt/server/steamcmd
+
+# SteamCMD fails otherwise for some reason, even running as root.
+# This is changed at the end of the install process anyways.
+chown -R root:root /mnt
+
+export HOME=/mnt/server
+./steamcmd.sh +login anonymous +force_install_dir /mnt/server +app_update 740 +quit
+
+mkdir -p /mnt/server/.steam/sdk32
+cp -v linux32/steamclient.so ../.steam/sdk32/steamclient.so
+EOF;
+
+        $this->option['csgo'] = ServiceOption::updateOrCreate([
+            'service_id' => $this->service->id,
+            'tag' => 'csgo',
+        ], [
+            'name' => 'Counter-Strike: Global Offensive',
+            'description' => 'Counter-Strike: Global Offensive is a multiplayer first-person shooter video game developed by Hidden Path Entertainment and Valve Corporation.',
+            'docker_image' => 'quay.io/pterodactyl/core:source',
+            'config_startup' => '{ "done": "VAC secure mode is activated.", "userInteraction": []}',
```

Comment:
I never noticed that, but I never had an issue with it erroring out. Will change it now.